telemetry/main.schema.json

```diff
@@ -868,6 +879,49 @@
         "$ref": "#/definitions/scalar"
       }
     },
+    "event": {
+      "type": "array"
+    },
+    "process": {
+      "type": ["object", "array"],
+      "properties": {
+        "scalars": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/scalar"
+          }
+        },
+        "keyedScalars": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/keyedScalar"
+          }
+        },
+        "histograms": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/histogram"
+          }
+        },
+        "keyedHistograms": {
+          "type": "object",
+          "additionalProperties": {
+            "additionalProperties": {
+              "$ref": "#/definitions/histogram"
+            }
+          }
+        },
+        "events": {
+          "type": "array",
+          "additionalProperties": {
+            "$ref": "#/definitions/event"
```

Comment:
Are you sure this makes it match the individual entries?
I'd expect this to properties to the "events" definition here.